lbz/authentication.py

```diff
@@ -15,7 +15,7 @@ def remove_prefix(text: str):
     """
     Removes prefix of a text based on : sign.
     """
-    return text[text.index(":") + 1 :] if ":" in text else text
+    return text[text.index(":") + 1 :] if ":" in text else text  # noqa:E203
```

Comment:
I'm wondering if sth like `re.sub(".*\:", "", text)` would be enough, instead of the function

Reply:
re module in python is quite an expensive operation. te noqa is only because of black.

Reply:
I agree with @pdyba, we should use regex only if there is no other option to handle the problem. The performance may be the argument but not only - regular expressions may cause some bigger unexpected problems as in [Cloudflare](https://blog.cloudflare.com/cloudflare-outage/) in 2019 ([sekurak](https://sekurak.pl/zabilo-cale-cloudflare-przerywamy-spotkanie-wszystkie-rece-do-konsol/) was writing about that too) 😉

Reply:
Interesting, although I think the regex I proposed is really straightforward and would not be that painful.

Reply:
They thought the same 😆  Anyway, still, the performance and probably that's not the right PR to change that.